site/docs/master/advanced-installation.md

```diff
@@ -1,83 +1,38 @@
-# Install Overview
+# Advanced installation topics
```

Comment:
Maybe instead of calling this page "advanced installation", we call it something like "customize your install"? WDYT?

Reply:
Agree. Renamed this to `Customize Install` and the file name as well.